pages/docs/advanced/react-native.en-US.mdx

```diff
@@ -0,0 +1,102 @@
+import Callout from 'nextra-theme-docs/callout'
+
+# React Native
+
+<Callout>
+  Upgrade to the latest version (≥ 1.0.0) to experience this customization.
+</Callout>
+
+Unlike React running inside the browsers, React Native has a very different use experience. For example there is no “tab focus”, switching from the background to the app is considered as a “focus” instead.
+To customize these behaviors, you can replace the default browser `focus` and `online` events listeners with React Native’s app state detection and other native ported API, and configure SWR to use them.
+
+
+## Example
+
+
+### Global Setup
+
+You can wrap your app under `SWRConfig` and preconfig all configurations there
+
+```jsx
+<SWRConfig 
+  value={{
+    /* ... */
+  }}
+>
+  <App>
+</SWRConfig>
+```
+
+### Pre-loaded Data
+
+You might want preload some data when app launches, these data are settled in SWR cache.
+
+```jsx
+<SWRConfig 
+  value={{
+    provider: () => new Map([
+      ['hello', 'swr'],
+      ['initialKey', 'initialValue']
+    ]),
+  }}
+>
+  <App />
+</SWRConfig>
+```
```

Comment:
I wonder if this is necessary for the RN docs, since we have a dedicated prefetching page. Also I think `fallback` is better for this case (so you can continue to use the global `mutate`)

Reply:
removed, make sense to let user checkout other doc section for details to decide what they need, preloaded state or fallback data